drivers/wifi/nrf700x/zephyr/src/qspi/inc/qspi_if.h

```diff
@@ -103,8 +103,9 @@ int func_rpu_sleep(void);
 int func_rpu_wake(void);
 int func_rpu_sleep_status(void);
 #endif /* CONFIG_NRF_WIFI_LOW_POWER */
-#endif /* __QSPI_IF_H__ */
 
 #define QSPI_KEY_LEN_BYTES 16
```

Comment:
Can the define be clubbed with the other defines at the top of the file?

Reply:
Its convenient and common practice to club defines with data structures.